src/Windowing/Silk.NET.Windowing.Common/Structs/WindowOptions.cs

```diff
@@ -120,26 +122,40 @@ public WindowOptions
             VSync = isVSync;
         }
 
+        static WindowOptions()
+        {
+            string name = "Silk.NET Window";
+            try
+            {
+                var assName = Assembly.GetEntryAssembly()?.GetName().Name;
```

Comment:
lmfao assName huehuehue he said ass

(generally i prefer `asm` instead of `ass`)